README.md

```diff
@@ -33,14 +33,10 @@ cargo install bpf-linker
 
 ### Using external LLVM
 
-On Debian based distributions you need to install the `llvm-19-dev`, `libclang-19-dev`
-and `libpolly-19-dev` packages. If your distro doesn't have them you can get them
-from the official LLVM repo at https://apt.llvm.org.
+On Debian based distributions you need to install the `llvm-21-dev`, `libclang-21-dev`
+and `libpolly-21-dev` packages, from the official LLVM repo at https://apt.llvm.org.
 
-On rpm based distribution you need the `llvm-devel` and `clang-devel` packages.
```

Comment:
The RPM packages are always one version behind and it's not different now - there is only 20 even in COPR repos. I think it's not worth mentioning them.